### ISSUE TITLE
brand_name in ConfigFile.plist

### ISSUE THREAD

**Issue by andyjriley** [LP-Messaging/iOS-Messaging-SDK#1]
The brand_name value is not replacing the "SDK Brand Name", this is also the case within the uploaded sample apps

**Comment by andyjriley** [comment]
Additional Info:
-Xcode 7.2.1 on 10.11.3 OSX
-Tried both Swift and ObjC variations

**Comment by adilusman1** [comment]
I am getting the same problem. Can we get it fixed please

**Comment by andyjriley** [comment]
Maybe a little more information will help. When first presenting the conversation view controller before there is an active conversation the title will be "SDK Brand Name", when the view is refreshed it will present the string from the bundle

**Comment by udidg** [comment]
Hi Andy, 

We use Swift 2.2 hence we support Xcode 7.3 only. 
Also Please provide more info: 
- Are you using latest version 1.1.106?
- iOS version?
- Are you using view controller mode or window mode (find detailed information in the integration document)
- Screenshots would defiantly be helpful.

**Comment by andyjriley** [comment]
I am using the latest in the master, which is 1.1.102, however pulled 1.1.105 out of the sample apps because it was latest version. I also tried using the interim branch which is currently 1.1.108. I have used both Objective C & Swift versions with 8.4 and 9.3 within the iOS Simulator.

This problem only arises when using the view controller mode on both iPhone & iPad.
These screenshots were built using Xcode 7.3 & 9.3 Simulator

![simulator screen shot apr 25 2016 10 11 53 am](https://cloud.githubusercontent.com/assets/5934581/14790834/4ad30d6c-0ad0-11e6-87ff-906fa98d67e4.png)
![simulator screen shot apr 25 2016 10 11 49 am](https://cloud.githubusercontent.com/assets/5934581/14790835/4aebff0c-0ad0-11e6-9c45-d6aa6f5f618d.png)
